app/src/components/main/scrollWrapper/scrollWrapper.jsx

```diff
@@ -50,6 +50,7 @@ export class ScrollWrapper extends Component {
     thumbMinSize: PropTypes.number,
     withBackToTop: PropTypes.bool,
     withFooter: PropTypes.bool,
+    withPaginationOffset: PropTypes.bool,
```

Comment:
May be ads this as a number to specify the 'contentMarginBottom'?
Or even better add the 'contentClassName' prop to provide any styles.

Reply:
Wouldn't it be better to keep contentMarginBottom, since it's used in many places with pagination to avoid repeating the same style? I can also add contentClassName for any additional specific styles